src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Utils/ModuleContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Bannerlord.LauncherManager;
+using Bannerlord.ModuleManager;
+using NexusMods.Games.MountAndBlade2Bannerlord.Models;
+
+namespace NexusMods.Games.MountAndBlade2Bannerlord.Utils;
+
+internal class ModuleContext
+{
+    private readonly IDictionary<string, LoadoutModuleViewModel> _lookup;
+    public ModuleContext(IEnumerable<LoadoutModuleViewModel> moduleVMs)
```

Comment:
I believe this code already exists in `Bannerlord.LauncherManager.Utils`, as `private class ModuleContext<TModuleViewModel> where TModuleViewModel : class, IModuleViewModel`.

Would it be better to instead make the above public instead of duplicating the code? I'm just curious here.

Reply:
Good point, made it public and renamed to `DefaultModuleContext`